components/esp_hw_support/port/esp32c2/sar_periph_ctrl.c

```diff
@@ -24,7 +25,7 @@
 
 static const char *TAG = "sar_periph_ctrl";
 
-static int rtc_spinlock;
+int rtc_spinlock;
```

Comment:
may I suggest to use `__` prefix in global symbols such as this lock?

Reply:
I think this will require changing code for other SoC's as well.
Should we do it afterwards in a separate PR?

Reply:
We agreed this change should go in a separate PR, as it affects other SoC's as well.